build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -35,16 +36,28 @@
     <Rule Id="CA1720" Action="None" />
     <Rule Id="CA1721" Action="None" />
     <Rule Id="CA1724" Action="None" />
+    <Rule Id="CA1801" Action="None" />
+    <Rule Id="CA1802" Action="None" /> <!-- disabled due to lack of support for string literals -->
+    <Rule Id="CA1804" Action="None" />
+    <Rule Id="CA1806" Action="None" />
```

Comment:
:memo: I'm *really* interested to see what the results of this were. Will have to enable it locally to experiment.

(The rule is **Do not ignore method results**)